rllib/connectors/action/pipeline.py

```diff
@@ -19,10 +21,17 @@
 class ActionConnectorPipeline(ConnectorPipeline, ActionConnector):
     def __init__(self, ctx: ConnectorContext, connectors: List[Connector]):
         super().__init__(ctx, connectors)
+        self.timers = defaultdict(_Timer)
```

Comment:
I thought about putting this in the ConnectorPipeline class but it has no functionality related to timers if I do that and it would make the inheritance model which we have here even weirder.